src/frontend/src/components/Split/index.tsx

```diff
@@ -106,6 +96,7 @@ interface Props {
  * functionality of our dev server because the config won't be rebuilt.
  */
 const SplitInitializer = ({ children }: Props): JSX.Element | null => {
+  const pathogen = useSelector(selectCurrentPathogen);
```

Comment:
It looks like -- for localdev -- the local config is basically locked to the values based on the originally loaded pathogen? I'm pretty sure what I'm seeing below will only prep and load the localdev config based on the pathogen the app is on during load. But if the pathogen changes, it will _not_ do any kind of new config, the config is basically locked until next browser refresh.

I know you're going to do another PR later to address the pathogen aspect more specifically, will that cover having the localdev config switch as part of pathogen switching? I could also believe that's just a huge hassle the way Split has things built and not really doable, but it would certainly be nice if we could.

Reply:
I will take this into consideration in the next PR!